### ISSUE TITLE
Rename "DataLab" to "Bennett Institute"

### ISSUE THREAD

**Issue by lucyb** [opensafely-core/job-server#3958]
Requires #3946 to be completed first.

Background 🧵 [here](https://bennettoxford.slack.com/archives/C04NHSU5DRB/p1704279098820819).

The DataLab organisation in Job Server should be renamed to Bennett Institute, given that we renamed ourselves a while ago.

We can rename the organisation without any negative effects (I believe), but we should update the URLs as well for consistency (represented by the `slug` field in the Organisation class). It's possible to do this within the staff area; this will automatically create a redirect as well. However, we could consider doing this as a migration, so that all other developers will have the change applied without needing a new copy of the production database.

This has the potential to cause some confusion, so we need to clearly communicate the change to everyone within the Bennett Institute.

**Comment by iaindillingham** [comment]
I was going to do this in a migration as part of #3946. I'll not do this work as part of that issue, and instead create a new branch/PR for this work.

**Comment by iaindillingham** [comment]
I asked for thoughts on this issue[^1]...

> I’m unclear whether I should or shouldn’t create a migration for the rename. On the one hand, using the admin interface creates a redirect. On the other hand, having used the admin interface, everyone developing job-server would have to fetch a new DB dump to see the changes locally. This wouldn’t be the case with a migration.

I think there was slightly more support for the admin interface approach than for the migration approach.

* It would be slightly less work for developers in the short term, but would introduce a small amount of work for a developer in the short term, too.
* Whilst both approaches could be tested locally, a migration wouldn't be as well tested as the code that underpins the admin interface.

[^1]: https://bennettoxford.slack.com/archives/C069SADHP1Q/p1704810336233969

**Comment by iaindillingham** [comment]
> we need to clearly communicate the change to everyone within the Bennett Institute

I think a message in #general would be enough. Something like:

> **job-server announcement:** We have renamed "DataLab" (the organisation) to "Bennett Institute". If you have bookmarked the former, then the URL will redirect to the latter automatically.

**Comment by iaindillingham** [comment]
@ghickman, I've noticed that whilst changing the slug adds a redirect for the main site, it doesn't add a redirect for the staff area. I think this is okay; the staff area isn't public-facing, and shouldn't be crawled. Whilst a user may visit the old URL because it's bookmarked/in their history, hopefully they'll realize that it has changed and navigate to the new URL. Do you agree?

**Comment by iaindillingham** [comment]
I've changed the name and slug on job-server, and tested that the redirect works on the main site. I've checked with Louis and Millie that the rename doesn't break anything (they don't think it does), and supplied a new dump that includes the rename (they're going to check that it doesn't). I've announced the rename on Slack.[^1] I also moved the snippet for creating a new dump from `DEVELOPERS.md` to `scripts/dump-co-pilot-reporting-data.sh`.

[^1]: https://bennettoxford.slack.com/archives/C31D62X5X/p1704891287094759
